org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/BackendUtils.java

```diff
@@ -1073,4 +1078,75 @@ public static void deriveWPD(TEntityType ci, List<String> errors) {
 		}
 		return res;
 	}
+
+	/**
+	 * Check the id's for new nodes, avoid repeating of target labels
```

Comment:
The general rule is that you should not use an apostrophe to form the plurals of nouns -- https://en.oxforddictionaries.com/punctuation/apostrophe